edx_repo_tools/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+__version__ = '0.3.3'
```

Comment:
We prefer that the final line in a file have a newline.  This repo (and many others) has a .editorconfig file. If you enable editorconfig in your IDE/editor, these things will be taken care of automatically.